internal/pkg/template/workload.go

```diff
@@ -751,12 +751,13 @@ type WorkloadOpts struct {
 	StateMachine       *StateMachineOpts
 
 	// Additional options for request driven web service templates.
-	StartCommand         *string
-	EnableHealthCheck    bool
-	Observability        ObservabilityOpts
-	Private              bool
-	AppRunnerVPCEndpoint *string
-	Count                *string
+	StartCommand              *string
+	EnableHealthCheck         bool
+	Observability             ObservabilityOpts
+	Private                   bool
+	AppRunnerVPCEndpoint      *string
+	Count                     *string
+	RuntimeEnvironmentSecrets map[string]Secret
```

Comment:
New variable RuntimeEnvironmentSecrets added.

Reply:
Can we remove `Runtime` throughout the PR? Or is there any reason why we want to add it?

Also why not reuse the existing `Secret` in `WorkloadOpts` for request driven web service (rdws)?

Reply:
Ack. I have reused Secret